src/Biconnectivity/articulation.jl

```diff
@@ -79,8 +79,10 @@ function articulation end
 
     articulation_points = Vector{T}()
 
-    for u in findall(is_articulation_pt)
-        push!(articulation_points, T(u))
+    for u in vertices(g)
+        if is_articulation_pt[u]
+            push!(articulation_points, u)
+        end
```

Comment:
Is there a way to do this in one statement, perhaps using `filter`?

Reply:
I don't think `filter` or `findall` support converting to a type, we could do a `findall` followed by a `convert`, but that would allocate more memory.

Reply:
where are we type-converting? Also, just noticed that `is_articulation_pt` is a bitvec, so...
```
articulation_points = vertices(g)[is_articulation_point]
```
should do it?